src/app/components/layout/header/header.component.scss

```diff
@@ -51,22 +51,22 @@ $img-height-small: 32px;
 }
 .nav-inner {
     display: block;
-    width: 100%;
+    width: 270px;
     height: 100%;
     background: #fff;
     transform: translateX(0);
     transition: transform 400ms ease-in-out;
     text-align: left;
-    overflow-y: auto;
```

Comment:
Removing this made the scroll stop working on mobile (Chrome and Firefox - Android).